bgs/bgs.go

```diff
@@ -152,6 +152,16 @@ func (bgs *BGS) StartDebug(listen string) error {
 	})
 	http.Handle("/prometheus", prometheusHandler())
 
+	http.HandleFunc("/feddbg/conns", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
What this endpoint does is self explanatory, but the path name and plan going forward are not. It's a new public web HTTP endpoint on a critical service, and I can't tell why it needs to be there.

A clearer path might be `/debug/upstream-conns`.

Reply:
```suggestion
	http.HandleFunc("/debug/upstream-conns", func(w http.ResponseWriter, r *http.Request) {
```